deepecohab/antenna_analysis/auxiliary_analysis.py

```diff
@@ -37,8 +37,7 @@ def calculate_approach_to_social_odor(
     results_path = Path(cfg['project_location']) / 'results' / 'results.h5'
     key='social_odor'
     
-    social_odor = None if overwrite else auxfun.load_ecohab_data(cfp, key, verbose=False)
-    
+    social_odor = None if overwrite else auxfun.load_ecohab_data(cfp, key)
```

Comment:
Let's get rid of the whole file. This does not belong in the main loop. Another repo will be created with deepecohab addons for analyses related to specific types of experiments.